lib/generator/template_values.rb

```diff
@@ -0,0 +1,48 @@
+module Generator
+  # Contains methods accessible to the ERB template
+  class TemplateValues
+    attr_reader :sha1, :version, :test_cases
+
+    def initialize(sha1:, version:, test_cases:)
+      @sha1 = sha1
+      @version = version
+      @test_cases = test_cases
+    end
+
+    def get_binding
+      binding
+    end
+  end
+
+  # This is not a standalone module, is that a problem?
```

Comment:
Comment is likely ready to be removed.  Or at least the question part of it, if it is for documenting/guidance.